api/src/test/java/org/openmrs/module/DaemonTokenTest.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by freddy on 20.05.18.
+ */
+public class DaemonTokenTest {
+	/*
+	**
+	* Check that DaemonTokenId is as it was set
+	*/
+	@Test
+	public void getId_shouldReturnIdSetInConstructor(){
+		
+		String tokenId = "DaemonTokenId12";
+		DaemonToken daemonToken = new DaemonToken(tokenId);
```

Comment:
we do not test constructors that simply set a field, I think this test should be removed.